File: app/src/main/java/com/tangpj/mvvmrecurve/MainActivity.java
package com.tangpj.mvvmrecurve;

import android.os.Bundle;
import android.widget.Button;

import com.tangpj.recurve.databinding.ActivityRecurveBinding;
import com.tangpj.recurve.databinding.ToolbarCollapsingRecurveBinding;
import com.tangpj.recurve.retrofit2.LiveDataCallAdapterFactory;
import com.tangpj.recurve.apollo.LiveDataApollo;
import com.tangpj.recurve.ui.activity.RecurveActivity;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends RecurveActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        ActivityRecurveBinding binding = initContentBinding(R.layout.activity_recurve);
    }
}

File: app/src/main/java/com/tangpj/mvvmrecurve/MainActivity.kt
package com.tangpj.mvvmrecurve

import android.os.Bundle
import com.tangpj.mvvmrecurve.databinding.CollasingTestBinding
import com.tangpj.mvvmrecurve.databinding.ContentTestBinding
import com.tangpj.recurve.ui.acitivty.RecurveActivity

class MainActivity: RecurveActivity(){

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val binding = initContentBinding<ContentTestBinding>(R.layout.content_test)
        appbar {
            scrollEnable = false
            toolBar {
                title = "测试测试"
            }
        }
    }
}